src/app/ASKS/ask/index.js

```diff
@@ -722,6 +722,23 @@ const Ask = ({
           clientId={clientId}
         />
       )}
+      {component === 'time' && (
```

Comment:
Instead of doing this you can do 

 {(component === 'text' || component === 'time) && (
        <Text.Write
          questionCode={questionCode}
          mandatory={mandatory}
          data={data}
          onSendAnswer={onSendAnswer}
          regexPattern={regexPattern}
          errorMessage={errorMessage}
          attributeCode={attributeCode}
          targetCode={targetCode}
          parentCode={parentCode}
          placeholderName={placeholderName}
          clientId={clientId}
          inputmask={inputmask}
        />
      )}

Reply:
The fields are not identical, so this will not work - the time component specifies the type of the <Input> component. The Read has been added to the attribute file

Reply:
You can add another prop and do the check for type inside of Text component.

{(component === 'text' || component === 'time) && (
<Text.Write
questionCode={questionCode}
mandatory={mandatory}
data={data}
onSendAnswer={onSendAnswer}
regexPattern={regexPattern}
errorMessage={errorMessage}
attributeCode={attributeCode}
targetCode={targetCode}
parentCode={parentCode}
placeholderName={placeholderName}
clientId={clientId}
inputmask={inputmask}
 type={component === 'time' ? 'time' : 'text'} or type={component === 'time' ? 'time' : null}
/>
)}